wrappers/kn/src/nonAndroidNativeTest/kotlin/Test.nonAndroidNative.kt

```diff
@@ -0,0 +1,34 @@
+import korlibs.io.file.std.LocalVfs
+import korlibs.io.posix.posixGetcwd
+import kotlinx.cinterop.*
+import kotlinx.coroutines.runBlocking
+import stb_image.STBI_grey
```

Comment:
Where does the stb_image package come from? I would have expected there needed to be a mentioning in the libs.version.toml (and other places)?

Reply:
See: https://github.com/zxing-cpp/zxing-cpp/pull/719#discussion_r1487027999

Reply:
Understood. See my suggestion from above regarding a minimal test.